src/index.html

```diff
@@ -1,15 +1,196 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+  action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+  method="POST"
+  autocomplete="off">
```

Comment:
FIx the indentations.
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      autocomplete="off"
    >
```